data/stoa0171/stoa001/stoa0171.stoa001.opp-lat3.xml

```diff
@@ -2,7 +2,9 @@
 <teiHeader>
     <fileDesc>
       <titleStmt>
-        <title>Corpus Scriptorum Ecclesiasticorum Latinorum (CSEL)</title>
+        <title xml:lang="lat">De Ave Phoenice</title>
+          <author xml:lang="lat">Lactantius</author>
+          <editor>Samuel Brandt</editor>
```

Comment:
You may add the second editor here if you like. He need not have his name tagged as "deu".